discord/client.py

```diff
@@ -448,7 +448,7 @@ async def on_error(self, event_method: str, *args: Any, **kwargs: Any) -> None:
         overridden to have a different implementation.
         Check :func:`~discord.on_error` for more details.
         """
-        print(f"Ignoring exception in {event_method}", file=sys.stderr)
+        print(f"Ignoring exception in {event_name}", file=sys.stderr)
         traceback.print_exc()
```

Comment:
Maybe this could be changed to `traceback.print_exception()` so that the `exception` parameter is actually used?